src/org/openstreetmap/josm/data/osm/DefaultNameFormatter.java

```diff
@@ -446,6 +452,12 @@ private static String getRelationTypeName(IRelation<?> relation) {
     private static String getNameTagValue(IRelation<?> relation, String nameTag) {
         if ("name".equals(nameTag)) {
             return formatLocalName(relation);
+        } else if ("from-to".equals(nameTag)) {
+            String from = trcLazy("from", I18n.escape(relation.get("from")));
+            String to = trcLazy("to", I18n.escape(relation.get("to")));
+            if (from != null || to != null)
+                return (from != null ? from : "?") + "-" + (to != null ? to : "?");
```

Comment:
Did you intend it to return `-<TO>` when `from` is `null` and `<FROM>-` when `to` is `null`?